doc/man5/x509v3_config.pod

```diff
@@ -122,242 +158,249 @@ following PKIX, NS and MS values are meaningful:
 include any email addresses contained in the certificate subject name in
 the extension.
 
-The IP address used in the B<IP> options can be in either IPv4 or IPv6 format.
+The IP address used in the B<IP> option can be in either IPv4 or IPv6 format.
 
-The value of B<dirName> should point to a section containing the distinguished
-name to use as a set of name value pairs. Multi values AVAs can be formed by
-prefacing the name with a B<+> character.
+The value of B<dirName> is specifies the configuration section containing
+the distinguished name to use, as a set of name-value pairs.
+Multiple values AVAs can be formed by prefacing the name with a B<+> character.
 
-otherName can include arbitrary data associated with an OID: the value
-should be the OID followed by a semicolon and the content in standard
-L<ASN1_generate_nconf(3)> format.
+The value of B<otherName> can include arbitrary data associated with an OID;
+the value should be the OID followed by a semicolon and the content in specified
+using the syntax in L<ASN1_generate_nconf(3)>.
 
 Examples:
 
- subjectAltName=email:copy,email:my@other.address,URI:http://my.url.here/
- subjectAltName=IP:192.168.7.1
- subjectAltName=IP:13::17
- subjectAltName=email:my@other.address,RID:1.2.3.4
- subjectAltName=otherName:1.2.3.4;UTF8:some other identifier
+ subjectAltName = email:copy, email:my@other.address, URI:http://my.url.here/
 
- subjectAltName=dirName:dir_sect
+ subjectAltName = IP:192.168.7.1
 
- [dir_sect]
- C=UK
- O=My Organization
- OU=My Unit
- CN=My Name
+ subjectAltName = IP:13::17
+
+ subjectAltName = email:my@other.address, RID:1.2.3.4
 
+ subjectAltName = otherName:1.2.3.4;UTF8:some other identifier
+
+ [extensions]
+ subjectAltName = dirName:dir_sect
+
+ [dir_sect]
+ C = UK
+ O = My Organization
+ OU = My Unit
+ CN = My Name
 
 =head2 Issuer Alternative Name
 
-The issuer alternative name option supports all the literal options of
-subject alternative name. It does B<not> support the email:copy option because
-that would not make sense. It does support an additional issuer:copy option
-that will copy all the subject alternative name values from the issuer
-certificate (if possible).
+This extension supports most of the options of subject alternative name;
+tt does not support B<email:copy>.
+It also adds B<issuer:copy> as an allowed value, which copies any subject
+alternative names from the issuer certificate, if possible.
 
 Example:
 
  issuerAltName = issuer:copy
 
-
 =head2 Authority Info Access
 
-The authority information access extension gives details about how to access
-certain information relating to the CA. Its syntax is accessOID;location
-where I<location> has the same syntax as subject alternative name (except
-that email:copy is not supported). accessOID can be any valid OID but only
-certain values are meaningful, for example OCSP and caIssuers.
+This extension gives details about how to retrieve information that
```

Comment:
This extensions gives details about how to retrieve information related to the certificate that the CA makes available.

Reply:
I don't understand this comment.

Reply:
*NOW* I understand it.